docs/AboutUs.md

```diff
@@ -3,7 +3,7 @@
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
 ![](https://via.placeholder.com/100.png?text=Photo) | Daniel | [Github](https://github.com/daniellimws) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Vishesh | [Github](https://github.com/wish2023) | [Portfolio](docs/team/johndoe.md)
+![](docs/team/vishesh-arora.jpg) | Vishesh | [Github](https://github.com/wish2023) | [Portfolio](docs/team/vishesh-arora.md)
```

Comment:
Not sure if the hyperlink will work cus it didnt work in mine. 
Try using `team/vishesh-arora.jpg`  instead of `docs/team/vishesh-arora.jpg`
And also the file name should be your github name not `vishesh-arora`
![image](https://user-images.githubusercontent.com/60333397/98456977-a70b4000-21bd-11eb-9051-8e443b77d5d5.png)